x-pack/plugins/observability/public/pages/alerts/components/alerts_search_bar.tsx

```diff
@@ -6,10 +6,12 @@
  */
 
 import { IndexPatternBase } from '@kbn/es-query';
-import { i18n } from '@kbn/i18n';
 import React, { useMemo, useState } from 'react';
 import { SearchBar, TimeHistory } from '../../../../../../../src/plugins/data/public';
 import { Storage } from '../../../../../../../src/plugins/kibana_utils/public';
+import { translations } from '../../config';
```

Comment:
@claudiopro CI fails because of this. I guess it should be `import { translations } from '../../../config';`